src/components/EquipmentItem.tsx

```diff
@@ -20,6 +20,8 @@ const EquipmentItem: FunctionComponent<EquipmentItemProps> = ({
 }) => {
   //it would be great for this to be a getter
   const itemName = item.manufacturer && item.model ?item.manufacturer + " " + item.model : item.description;
+  //if there currently is no quantity, set the selected quantity of the item to 0
+  quantity = quantity | 0;
```

Comment:
In the parameters you can provide a default `EquipmentItem = ({ quantity = 0 }) => `, and you've also told TypeScript you expect quantity to be a number (not undefined, not null).  So why do we need to do this?
```suggestion
```